dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/ArtifactsTest.java

```diff
@@ -34,5 +34,5 @@ public void testToCoordinates_withPackaging() {
     Artifact artifact = new DefaultArtifact("com.google.example", "example", "jar", "1.3.2");
     Assert.assertEquals("com.google.example:example:1.3.2", Artifacts.toCoordinates(artifact));
   }
-
+  
```

Comment:
Undo whitespace regression?

Reply:
https://groups.google.com/a/google.com/forum/#!msg/java-users/_ABiES5o020/J3TVUclzN2gJ

Reply:
Official Google Java code review policy is to never comment on trailing white space. Critique doesn't even show it. Unfortunately Github does, but we should still not worry about it.